pkg/capabilities/capabilities.go

```diff
@@ -82,7 +83,10 @@ func (c *Capabilities) initialize(bypass bool) error {
 	}
 
 	for c := range c.all {
-		cap.DropBound(c) // drop all capabilities from bound
+		err = cap.DropBound(c) // drop all capabilities from bound
+		if err != nil {
+			logger.Error("Dropping capability from bound", "cap", c, "error", err)
```

Comment:
This is Debug or even Info level.
It is not critical and expected to fail in many runs.

Reply:
As expected to fail, Debug.